diff --git a/node_modules/react-native-push-notification/android/.project b/node_modules/react-native-push-notification/android/.project
index 3865e0f..3964dd3 100644
--- a/node_modules/react-native-push-notification/android/.project
+++ b/node_modules/react-native-push-notification/android/.project
@@ -5,11 +5,6 @@
 	<projects>
 	</projects>
 	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
 		<buildCommand>
 			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
 			<arguments>
@@ -17,7 +12,6 @@
 		</buildCommand>
 	</buildSpec>
 	<natures>
-		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>org.eclipse.buildship.core.gradleprojectnature</nature>
 	</natures>
 </projectDescription>
diff --git a/node_modules/react-native-push-notification/android/build.gradle b/node_modules/react-native-push-notification/android/build.gradle
index d6019f6..f14f66c 100644
--- a/node_modules/react-native-push-notification/android/build.gradle
+++ b/node_modules/react-native-push-notification/android/build.gradle
@@ -50,15 +50,14 @@ android {
 }
 
 dependencies {
-    // Use either AndroidX library names or old/support library names based on major version of support lib
-    def supportLibVersion = safeExtGet('supportLibVersion', '27.1.1')
-    def supportLibMajorVersion = supportLibVersion.split('\\.')[0] as int
-    def appCompatLibName =  (supportLibMajorVersion < 20) ? "androidx.appcompat:appcompat" : "com.android.support:appcompat-v7"
-
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     testImplementation 'junit:junit:4.12'
-    implementation "$appCompatLibName:$supportLibVersion"
+
+    implementation "androidx.appcompat:appcompat:1.6.1" // ✅ use AndroidX explicitly
     implementation 'com.facebook.react:react-native:+'
     implementation 'me.leolin:ShortcutBadger:1.1.22@aar'
-    implementation "com.google.firebase:firebase-messaging:${safeExtGet('firebaseMessagingVersion', '21.1.0')}"
+
+    implementation ('com.google.firebase:firebase-messaging:23.0.0') {
+        exclude group: 'com.android.support'
+    }
 }
